chainer/training/triggers/once_trigger.py

```diff
@@ -0,0 +1,36 @@
+class OnceTrigger(object):
+
+    """Trigger based on the starting point of the iteration.
+
+    This trigger accepts only once at starting point of the iteration. There
+    are two ways to specify the starting point: only starting point in whole
+    iteration or recalled when training resumed.
+
+    Args:
+        recall_on_resume (bool): Whether the extension is recalled or not when
```

Comment:
How about naming it `call_on_resume`? I'm wondering if "recall" could mean "call again".

Reply:
OK, I will change this argument.